waku/persistence/sqlite/sqlite.go

```diff
@@ -32,10 +34,35 @@ func addSqliteURLDefaults(dburl string) string {
 
 func executeVacuum(db *sql.DB, logger *zap.Logger) error {
 	logger.Info("starting sqlite database vacuuming")
-	_, err := db.Exec("VACUUM")
-	if err != nil {
-		return err
+
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
This code is exact duplicate of the code for postgres. Wondering if we could somehow have this function at a higher layer and call it from both these places? Especially since i don't see any db specific dependencies other than the actual command to execute which can be passed as an argument.

Reply:
Good idea. I ended up implementing a function that can be used for any slow query to log a message